docs/howto/run_material_mapping.md

```diff
@@ -0,0 +1,119 @@
+Four type of surfaces exist :  
+*  boundaries which represent the boundaries of the different volumes
+*  approaches which to respond to the entrance and exist of the detector layer
+*  representing which correspond to the surface representation of a layer (often positioned at the middle of the 2 approaches)
+*  sensitives which correspond to the active part of the detector (sensors)
+
+By default, all the surfaces will be written but one can turn a specific type off (for example the sensitive) by using the appropriate option : `mat-output-XXX false`
+
+The Json file can now be edited to select which surfaces and volumes you want to have material mapped on. The Json file contain a list of volumes, in those volumes you can find boundaries and layers, the layers then contains approaches, representing and sensitives surfaces. Information of the surface such as their type, range, id and position are available. To add one surface to the material mapping one simply need to switch the `mapMaterial` variable to true, the binning can then be change by changing the number associated to `bin0` and `bin1` the type of bin can also be change. For the volume the same method can be applied except that up to 3 bins can be associated.
+
+When mapping onto a surface, the material inside volumes with material (or ProtoMaterial) will be ignored, you should thus avoid mapping material onto surface within material volumes. When mapping onto a volume, only the material within that volume will be used. If you have a large gap between you last material surface and your volume you might then want to also map material onto the boundary of the material volume.
+
+
+This configuration can be cumbersome to do by hand especially when trying to map on sensitives surfaces. To simplify this task two python script are available in Examples/scripts/MaterialMaping :
+
+*  writeMapConfig.py
+*  configureMap.py
+
+The first one take as an input the surfaces map previously generated and will return a json config file. In this file you can see all the different type of surfaces associated to each volume. You can then change the binning associated to a surface type, when the second script is called the resulting map will have the binning applied to all the surface of the corresponding type and `"mapMaterial"` will be change to true. Attention : the configureMap.py will modify the surfaces map used in input you might want to make a copy of it beforehand.
+
+
+```
+python3 ../acts-framework/scripts/MaterialMaping/writeMapConfig.py geometry-map.json config-map.json
```

Comment:
```suggestion
../acts-framework/scripts/MaterialMaping/writeMapConfig.py geometry-map.json config-map.json
```

Reply:
I am not sure why you would remove the python3 from that line ?

Reply:
In the snippet below you don't have it. I assume the script has a shebang. That's all.

Reply:
The one below was in fact me forgetting... 
So lets keep it like that.